packer/ansible/roles/lvm-init/tasks/main.yml

```diff
@@ -0,0 +1,69 @@
+---
+- name: Wait until disk will be presend
+  wait_for:
+    path: /dev/xvdb
+
+- name: Create Volume Group
+  lvg:
+    vg: DataVG
+    pvs: "/dev/xvdb"
+
+- name: Create Thin Pool
+  register: thin_pool
+  failed_when: "thin_pool is failed and 'Sorry, no shrinking of DataLV to 0 permitted' not in thin_pool.msg"
+  lvol:
+    lv: DataLV
+    vg: DataVG
+    size: 100%FREE
+    opts: --thinpool ThinPool -V 1G
+
+- name: Format LVM
+  filesystem:
+    fstype: xfs
+    dev: /dev/DataVG/DataLV
+    opts: -L DATA
+
+- name: Mount
+  mount:
+    name: "/var/run/docker"
+    src: LABEL=DATA
+    fstype: xfs
+    opts: defaults,nofail
```

Comment:
According to the docs, this means that the boot will continue even if this mount point is not mounted successfully. I think we'd want to fail if the mount was unsuccessful.